helm/conjur-app-deploy/charts/app-secretless-broker/templates/test_app_secretless_broker.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: test-app-secretless-broker
+  labels:
+    app: test-app-secretless-broker
+spec:
+  ports:
+  - protocol: TCP
+    port: 8080
+    targetPort: 8080
+  selector:
+    app: test-app-secretless-broker
+  type: {{ .Values.global.appServiceType }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: test-app-secretless-broker
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: test-app-secretless-broker
+  name: test-app-secretless-broker
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: test-app-secretless-broker
+  template:
+    metadata:
+      labels:
+        app: test-app-secretless-broker
+    spec:
+      serviceAccountName: test-app-secretless-broker
+      containers:
+      - image: {{ printf "%s:%s" .Values.app.image.repository .Values.app.image.tag }}
+        imagePullPolicy: {{ .Values.app.image.pullPolicy }}
+        name: test-app
+        ports:
+        - name: http
+          containerPort: 8080
+        readinessProbe:
+          httpGet:
+            path: /pets
+            port: http
+          initialDelaySeconds: 15
+          timeoutSeconds: 5
+        env:
+          - name: DB_URL
+            value: postgresql://localhost:5432/test_app
+      - image: {{ printf "%s:%s" .Values.secretless.image.repository .Values.secretless.image.tag }}
+        imagePullPolicy: {{ .Values.secretless.image.pullPolicy }}
+        name: secretless
+        args: ["-f", "/etc/secretless/secretless_config.yaml"]
+        ports:
+        - containerPort: 5432
+        env:
+          - name: MY_POD_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+          - name: MY_POD_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: MY_POD_IP
+            valueFrom:
+              fieldRef:
+                fieldPath: status.podIP
+          - name: CONJUR_AUTHN_LOGIN
```

Comment:
If you make the change to `conjur_authn_configmap.yaml` suggested above, then you can delete this `-name: CONJUR_AUTHN_LOGIN` entry in the `env:` section, and simply add an entry in `envFrom` for that ConfigMap.